src/blocks/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+
+  margin-right: 4px;
+
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
Here it is better to add styling not only for 4 stars, but for everyone else.
![image](https://user-images.githubusercontent.com/105153156/215102624-7574358e-5a57-4697-ad9b-3387f9da2dca.png)